.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "third_party/duckdb"]
 	path = third_party/duckdb
 	url = https://github.com/duckdb/duckdb.git
-	hash = 17d598fc4472c64969f47f86c30fce75c4d64ed4
+	hash = fa5c2fe15f3da5f32397b009196c0895fce60820
```

Comment:
I didn't know `hash` property was a thing? And I don't see it in the [doc](https://git-scm.com/docs/gitmodules/en) either? Does that do anything?

Reply:
Oh my, it seems it may not, though it _seems_ to have some effect, that could be some other problem. Looking at the history, Thijs added this when we pinned circa 1.0.

I do really dislike git submodules. We could just put the duckdb directory into gitignore and have this all handled by the Makefile instead... food for thought.

Reply:
ok I just confirmed that on a clean checkout then `git submodule update --init --recursive` this appears to do nothing.

Reply:
changed PR to remove this instead